CHANGELOG.md

```diff
@@ -1,59 +1,61 @@
-openvpn Cookbook CHANGELOG
-==========================
+# openvpn Cookbook CHANGELOG
+
 This file is used to list changes made in each version of the openvpn cookbook.
 
+## v3.0.0
```

Comment:
Yup, if you can get me a list of the changes. I'll be happily to add them to the list. 
Because we didn't squash 'n merge the changes each commit didn't really relate to a PR :(

Reply:
A good maintainer is able to go over the commit log, diffs on each file, issues, PRs etc. to be able to collate a human-readable changelog. If we don't want to spend time on that, thats ok but lets add a better disclaimer here.

Reply:
```
Version 3 includes numerous re-factors and backwards-breaking changes; it's usage is different to the previous major release. The `README.md` should be used be consulted for any users testing an upgrade scenario.

For a full list of changes please see the [github-diff](https://github.com/sous-chefs/openvpn/compare/v2.1.0...master)

Anecdotally:

- Update of testing suites
- A lot of minor bug fixes
```